bazel/repositories.bzl

```diff
@@ -358,6 +359,13 @@ def _com_google_googletest():
         actual = "@com_google_googletest//:gtest",
     )
 
+def _com_github_abseil_abseil_cpp():
+    _repository_impl("com_github_abseil_abseil_cpp")
```

Comment:
Minor nitpick: Abseil is a native Bazel project, so we should use its declared workspace name here. Otherwise you'll get a warning at build time.

https://github.com/abseil/abseil-cpp/blob/master/WORKSPACE

_repository_impl("com_google_absl")